benchs/bench_all_ivf/bench_all_ivf.py

```diff
@@ -1,10 +1,10 @@
+#!/usr/bin/env python2
```

Comment:
python3, maybe do a pass of search/replace. Most files are compatible with both, but python3 should be the default.

Reply:
As above, we can do this in a subsequent PR.